src/client/extension.ts

```diff
@@ -205,6 +229,18 @@ function didOpenTextDocument(document: vscode.TextDocument): void {
             htmlServerRunning = true;
             _context.subscriptions.push(disposable);
         }
+
+        // this is used to send htmlServer telemetry events
+        client.onReady().then(() => {
+            client.onNotification("telemetry/event", (payload: string) => {
```

Comment:
it looks like this is copy/pasted from above in this file. Use a shared function to handle it and share the code.